msticpy/nbtools/nbdisplay.py

```diff
@@ -203,59 +230,123 @@ def display_logon_data(
          explicitly specify os_family (Linux or Windows)
          (the default is None)
 
+    """
+    display(format_logon(logon_event, alert, os_family))
+
+
+@export
+def format_logon(
+    logon_event: Union[pd.DataFrame, pd.Series],
```

Comment:
This looks like it only processes Linux Audit logons events and not Syslog events - you might want to reference this in the doc string.

Reply:
Added note to say only Windows logons supported currently (added in later branch)